libcontainer/cgroups/stats.go

```diff
@@ -121,6 +148,7 @@ type HugetlbStats struct {
 
 type Stats struct {
 	CpuStats    CpuStats    `json:"cpu_stats,omitempty"`
+	CpusetStats CpusetStats `json:"cpuset_stats,omitempty"`
```

Comment:
Ideally

```suggestion
	CPUSetStats CPUSetStats `json:"cpuset_stats,omitempty"`
```

But I can see that being inconsistent with the other field; is that something that can still be changed, or would that break many things?